### DISCUSSION TITLE
Configurable kickstart, pressed, unattended files

### DISCUSSION THREAD

**Discussion by sestegra** [vmware/packer-examples-for-vsphere#104] [Ideas]
Hello,

As kickstart, pressed, unattended files are built from a template file with Packer variables (currently addressing the locales), I suggest to address some options such as:
- DHCP or static IP, a common use case is a vSphere cluster that use an IPAM for assigning IP addresses
- Partition (LVM or not)

Regarding the static IP option, I found already raised issues.
- #16
- #17

What do you think about it?

Kind regards,
Stéphane

**Comment by tenthirtyam** [comment]
Hi Stéphane,

Yes, indeed - this h just so happens to be a couple of the items that I already have in the backlog (of which I need to stack rank). In fact, for the partitioning, my goal is to have that align with CIS. In fact, I was just discussing possible approaches, level-of-effort, and ranking with another individual that uses the project.

Let's revisit this end of week or early next week. I'd like to get things tidied up and a final set of test before cutting a v21.10 release.

If you are a member of the VMware {code} Slack workspace, you can find me as @tenthirtyam, for more real-time discussions.

Ryan

**Reply by sestegra** [reply]
I just connected on the VMware {code} Slack workspace to discuss this topic (find me as @sestegra).
So, let's connect to discuss when you're available.

Then, take the time you need to finalize the v21.10 release, this topic is only to discuss a possible backlog. For urgent topic, I create Issues.
